### ISSUE TITLE
PolarArray double-click editing doesn't work

### ISSUE THREAD

**Issue by DeepSOIC** [DeepSOIC/Lattice2#31]
- start FreeCAD, make a new document, change to lattice2
- click on icon PolarArray --> a polar array is created
- double click on the obj in tree view
- click on ok
- error message ...

```
Traceback (most recent call last):
  File "/home/hugo/.FreeCAD/Mod/lattice2/lattice2AttachablePlacement.py", line 93, in unsetEdit
    if not (mode == EDIT_ATTACHMENT or (mode == 0 and always_edit_attachment)): raise NotImplementedError()
<class 'NameError'>: name 'always_edit_attachment' is not defined
```

reported by bernd:
https://forum.freecadweb.org/viewtopic.php?p=320316#p320316

**Comment by DeepSOIC** [comment]
Turns out it has been fixed already, b315aeaaf6939454d48458ea1c0180a6369cde7c

**Comment by DeepSOIC** [comment]
The fix mentioned was for opening for edit. This bug is about closing the edit. DOH!
